projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -855,6 +883,30 @@ function wpcom_launchpad_get_newsletter_subscriber_count() {
 	return (int) $total_subscribers;
 }
 
+/**
+ * Determines if Stripe has been connected.
+ *
+ * @return bool Whether or not Stripe account is connected.
+ */
+function wpcom_is_stripe_connected() {
+	require_lib( 'memberships' );
```

Comment:
Yes, I was just asking about this on slack. The require_lib() method will always exist in simple/atomic environments, but certain libraries may not. Someone thought memberships may not be present by default on atomic, but I'm not sure how that could be possible given how much other code depends on it. 

I don't have strong feelings one way or the other for require_lib(), but it is very widely used in our codebases, so I've assumed it is acceptable, or even preferred, for many. But I'm happy to adjust.

Reply:
@daledupreez - I've updated this so we're only checking membership settings on simple for now. Checking the membership settings on atomic seems like it will require some extra work. 

I'm still using require_lib( 'memberships' ) as it still seems like the most straightforward and common way to pull in the needed methods. Let me know if you have an alternative approach in mind.

Reply:
Just to clarify, my dislike of `require_lib()` stems from the fact that it effectively masks the underlying dependency for a casual reader, making the dependency somewhat implicit, unless the reader happens to recall exactly how `require_lib()` does lookups. In this particular case, I can't tell what `'memberships'` is, so I can't easily tell what code we're expecting to be present (and is not present, in some cases!).

As a proposed alternative, I always prefer an explicit `require_once`, as it is generally 100% explicit as to what you're trying to pull in. In this specific case, I would go for something like:
```php
	require_once WP_CONTENT_DIR . '/lib/memberships/0-load.php';
```
To me, it would have been way easier to detect a possible problem with the expectations for this more explicit `require_once` approach, and very clear as to how we'd detect whether the files are present. 😁

Reply:
@daledupreez - I do understand your point. I had to do a fair amount of research today to understand how require_lib() works, where it searches for libraries, how it loads them, and when it's safe to use. By comparison, a require statement is simple and direct.

I wasn't quite sure how to include the library and associated methods without it, but you're right that including that 0-load file should do it. That's basically what require_lib() does under the hood. I'd be happy to update in a follow up.